lib/Crypto/Encryption.php

```diff
@@ -94,10 +94,17 @@ class Encryption implements IEncryptionModule {
 	private $decryptAll;
 
 	/** @var int unencrypted block size if block contains signature */
-	private $unencryptedBlockSizeSigned = 6072;
```

Comment:
I am still worried about this one TBH

Reply:
@DeepDiver1975 @butonic any educated guess here why was it set to 6072 ?

Reply:
"Legacy base64 encoding reduces unencrypted block size to 3/4"

Reply:
Unless I'm missing something, both the `unencryptedBlockSizeSigned` and `signatureSize` aren't assigned anywhere outside here, and they're only used inside the `getUnencryptedBlockSize` method.

I think it's better to move these variables inside the method as local variables, and add proper explanation there.

As for my reason not to use a constant, in this case, I think it's better to have the explanation of these values in only one place (inside the method that will use them) specially if there are no plans to use them anywhere else.

Reply:
I guess I am finally able to find the logic of that magic numbers:

```
base64 encode loss coefficient = 3/4
oc block size`= 8192

padding for unsigned block = "xx" (2 bytes)
padding for signed block`= "xxx" (3 bytes)

iv = "00iv00"(6 byte) + iv(16 byte) = 22 bytes
sign = "00sig00"(7 byte) +  sign(64 byte) = 71 bytes

Unencrypted block size for binary encoded unsigned file = (blockSize - iv - padding) = 8192-22-2 = 8168 bytes

Unencrypted block size for binary encoded signed file = (blockSize -sign - iv - padding) = 8192-71-22-3 = 8096 bytes

Unencrypted block size for base64 encoded unsigned file = (blockSize - iv - padding)*loss_coefficient = 6126 bytes

Unencrypted block size for base64 encoded signed file = (blockSize -sign - iv - padding)*loss_coefficient = 6072 bytes
```